### DISCUSSION TITLE
serverClientId is not supported on Web.

### DISCUSSION THREAD

**Discussion by Ekkas** [serverpod/serverpod#692] [Q&A] [answered]
Hi everyone,
I'm using Windows, android studio and trying to get the Web client to Google (and email) Authenticate.
However I get:
params.serverClientId == null
"serverClientId is not supported on Web." in the Flutter app when clicking on Google.

The google_client_secret.json file does have: 
"web":{"client_id":"62...apps.googleusercontent.com
and in sign_in_page.dart is set:
const _googleServerClientId = '62...apps.googleusercontent.com';

I believe the GoogleCloud setting is correct, allowing http://localhost:5000 and the flutter app runs on Chrome with --web-port=5000

Thank you
Anton

**Comment by yahu1031** [comment, marked as answer]
Check out this good playlist of serverpod [![Serverpod playlist](https://i.ytimg.com/vi/8dDucblphek/hq720.jpg?sqp=-oaymwExCK4FEIIDSFryq4qpAyMIARUAAIhCGAHwAQH4Af4JgALQBYoCDAgAEAEYEyA4KH8wDw==&rs=AOn4CLCVPEldiwX9kx0aelOoG6aq15G9pw)](https://youtube.com/playlist?list=PL0CMCjqejDDQFDNDCNJHGDnmwSbFpUWxq)

**Reply by Ekkas** [reply]
Thank you,

1) Turns out in sign_in_page.dart (auth example) the lines for Web(at least) should/could be:
const _googleClientId = '62....
const _googleServerClientId = null;

2) In main.dart, the line :
client = Client( 'http://localhost:5050/', (port for local web server)
As shown in https://www.youtube.com/watch?v=y7oLNlbWbUY
should be :
client = Client( 'http://localhost:8080/', (port for ServerPod Webserver?)

At least Google Sign-in on Web works for me like that.
Thank you!
Ekkas
